packages/react-vapor/src/components/input/Label.tsx

```diff
@@ -9,18 +9,18 @@ export interface ILabelProps {
     invalidMessage?: string;
 }
 
-export class Label extends React.Component<ILabelProps, any> {
+export class Label extends React.Component<ILabelProps & React.HTMLAttributes<HTMLLabelElement>, any> {

```

Comment:
You can remove the `htmlFor` declaration from the `ILabelProps` since it is already included in the type `React.HTMLAttributes<HTMLLabelElement>`